lib/WUI/http/httpd_post.c

```diff
@@ -354,7 +354,7 @@ void httpd_post_finished(void *connection,
     if (connection != NULL && status_code == 200) {
         strncpy(response_uri, endpoints[http_post_uri_file_index], response_uri_len);
     } else {
-        snprintf(response_uri, response_uri_len, "/%ld", status_code);
+        snprintf(response_uri, response_uri_len, "/%" PRIu32, status_code);
```

Comment:
this is a nice example of formatting macros, please report to the rest of the team :+1:

Reply:
Report where? Meeting? Some channel? Yes, I'd like to set up some kind of quality guild, but I haven't gotten around that yet.